### DISCUSSION TITLE
Custom Play / Stop Button

### DISCUSSION THREAD

**Discussion by memdigital** [katspaugh/wavesurfer.js#3597] [General]
Hello,

New to the plugin, and it is amazingly simple to use, but I wondered if there is a way to create a HTML link and/or button, that would start and stop the audio?

I would live to create my own buttons on a variety of websites.

Thanks Kindly

**Comment by jagthedrummer** [comment]
This example should point you in the right direction: https://wavesurfer.xyz/examples/?events.js

**Comment by ManimMadhav** [comment]
Here's a template of the code you can follow:

HTML code to create a button:
```
<button type="button" id="btn-playpause">Play/Pause</button>
```
JavaScript code to play or pause the audio using the button:
```
const play = document.getElementById('btn-playpause');
play.addEventListener('click', () => {
    wavesurfer.playPause();
});
